File: src/test/java/com/learn/student/StudentTest.java
package com.learn.student;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class StudentTest {
    @Test
    public void StudentScoreTest(){
        Student student = new Student("Raku",60,90);
        Assertions.assertEquals(90,student.highest());
    }

    @Test
    public void AverageTest(){
        Student student = new Student("Raku",60,80);
        Assertions.assertEquals(70,student.getAverage());
    }
}

File: src/test/kotlin/com/learn/StudentTest.kt
package com.learn

import com.learn.student.Student
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class StudentTest(){
    @Test
    fun highTest(){
        val stu = Student("Raku",60,80)
        Assertions.assertEquals(80,stu.highest())
    }
    @Test
    fun average(){
        val stu = Student("Raku",60,80)
        Assertions.assertEquals(70,stu.getAverage())
    }
    @Test
    fun gradingTest(){
        val stu = Student("Raku",60,80)
        Assertions.assertEquals('C',stu.grading())
    }
}